image_transport/include/image_transport/publisher_plugin.hpp

```diff
@@ -95,6 +103,18 @@ class PublisherPlugin
     publish(*message);
   }
 
+  /**
+   * \brief Publish an image using the transport associated with this PublisherPlugin.
```

Comment:
Do you mind to add in the `README,md` a comment related with this ?

Reply:
There is no README.md, should I create one? or add the comment to `mainpage.dox`?